api/ocm/extensions/repositories/genericocireg/annotation_test.go

```diff
@@ -63,6 +64,9 @@ var _ = Describe("Transfer handler", func() {
 		Expect(v.GetDescriptor()).To(YAMLEqual(`
   mediaType: application/vnd.oci.image.manifest.v1+json
   schemaVersion: 2
+  annotations:
```

Comment:
this should not be part of this PR as I mentioned in our sync as now the title and scope of the issue no longer fit IMO

Reply:
I suggest you draw these commits out again and create a separate PR, its not necessary to build on this one